src/components/button.js

```diff
@@ -3,18 +3,21 @@
 /* eslint-disable object-curly-newline */
 /* eslint-disable react/jsx-curly-spacing */
 /* eslint-disable react/require-default-props */
+/* eslint-disable react/prop-types */
+
 import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = ({
-  name, btn, color, wide,
+  name, btn, color, wide, clickHandler,
```

Comment:
Instead of disabling the linter, can you add `clickHandler` to the `PropTypes` instead? Seems it's a function, so consider adding a function type for it. see https://reactjs.org/docs/typechecking-with-proptypes.html for example and more explanation.